packages/realm/binding/jsi/flush_ui_queue_workaround.cpp

```diff
@@ -0,0 +1,62 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2024 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#include "flush_ui_queue_workaround.h"
+
+#include <ReactCommon/CallInvoker.h>
+
+namespace realm::js::flush_ui_workaround {
```

Comment:
As part of https://github.com/realm/realm-js/pull/6718 I did a refactor of the "flush UI workaround" which I decided to keep in this PR.